Documentation/api-reference.md

```diff
@@ -115,24 +119,27 @@ sendEdgeEvent: function(xdmData as object, callback = _adb_default_callback as f
 - `@param [optional] context as dynamic : context to be passed to the callback function`
 
 > This function will automatically add an identity property, the Experience Cloud Identifier (ECID), to each Edge network request within the Experience event's `XDM IdentityMap`.
+> Also `ImplementationDetails` are automatically collected and are sent with every Experience Event. If you would like to include this information in your dataset, add the "Implementation Details" field group to the schema tied to your dataset.
```

Comment:
Do we need to provide this context with the API description?

Reply:
Sounds good!